### PR TITLE
feat(rxApp): FRMW-953 Allow routes to be updated via an event

### PR DESCRIPTION
**Pull request by halkon**
JIRA:

### LGTMs
- [x] Dev LGTM
- [X] Dev+Vidyo LGTM
- [X] QE LGTM

### CODE REVIEW

File: src/components/rxApp/scripts/rxApp.js [hunk 1]
```diff
@@ -67,9 +67,14 @@ angular.module('encore.ui.rxApp')
...
-                scope.routes = routes;
-            });
+            var setRoutes = function () {
+                appRoutes.getAll().then(function (routes) {
+                    scope.routes = routes;
+                });
+            };
+
+            scope.$evalAsync(setRoutes);
+            scope.$on('oriUpdateNavRoutes', setRoutes);
```
Comment (CITguy) [inline src/components/rxApp/scripts/rxApp.js]: I would use an `rx` prefixed event name instead (`rxUpdateNavRoutes`).

**Comment by halkon** [comment]
added tests

**Comment by CITguy** [comment]
LGTM, please squash

**Comment by halkon** [comment]
done

**Pull request merged**